CSS/album.css

```diff
@@ -0,0 +1,118 @@
+/*CSS reset*/
```

Comment:
First of all, you must change the fold to lowercase.

Reply:
the folder has been changed to lowercase